src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -36,7 +34,8 @@
 import static uk.gov.hmcts.reform.sscs.reference.data.model.HearingPriority.URGENT;
 import static uk.gov.hmcts.reform.sscs.reference.data.model.HearingTypeLov.SUBSTANTIVE;
 
-@SuppressWarnings({"PMD.GodClass"})
+
+@SuppressWarnings({"PMD.GodClass","PMD.UnnecessaryLocalBeforeReturn", "PMD.ReturnEmptyCollectionRatherThanNull", "PMD.GodClass", "PMD.ExcessiveImports",  "PMD.UseConcurrentHashMap"})
```

Comment:
I don't think we should be adding more suppressions

Reply:
Got rid of extra suppressions that were added, few of them were a result of incorrect rebase and implemented suggestions for others.